### PR TITLE
fix: Resolves English leaks within safe-mode

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:

- Resolve English leaks within safe-mode flows

### Screenshot/Video:

Screen Shot 2021-05-03 at 11 00 32 AM

### Reviewers

- @mauriciocastillosilva-okta 

### Issue:

- [OKTA-388435](https://oktainc.atlassian.net/browse/OKTA-388435)
- [OKTA-388441](https://oktainc.atlassian.net/browse/OKTA-388441)
- [OKTA-388446](https://oktainc.atlassian.net/browse/OKTA-388446)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -116,9 +116,6 @@ module.exports = {
             'playground/**/error-unlock-account.json',
             'playground/**/error-user-is-not-assigned.json',
             'playground/**/identify-unknown-user.json',
-            'playground/**/safe-mode-credential-enrollment-intent.json',
```
Comment (nikhilvenkatraman-okta) [inline .eslintrc.js:119]: Thank you for removing them from this list

File: playground/mocks/data/idp/idx/safe-mode-credential-enrollment-intent.json [hunk 2]
```diff
@@ -11,9 +11,9 @@
     "type": "array",
     "value": [
       {
-        "message": "Set up is temporarily unavailable due to server maintenance. Click 'Back to Settings' to continue.",
+        "message": "Set up is temporarily unavailable due to server maintenance. Try again later.",
         "i18n": {
-          "key": "idx.error.server.safe.mode.cred.enroll"
+          "key": "idx.error.server.safe.mode.enrollment.unavailable"
```
Comment (jmelberg-okta) [inline playground/mocks/data/idp/idx/safe-mode-credential-enrollment-intent.json:16]: Should have mentioned this in the PR description, but all safe-mode errors include a single i18n key. You can see the server-side implementation [here](https://github.com/okta/okta-core/pull/47490/files).

**Review by nikhilvenkatraman-okta — APPROVED** [review]
🚀

File: test/unit/spec/v2/ion/responseTransformer_spec.js [hunk 3]
```diff
@@ -71,7 +71,7 @@ describe('v2/ion/responseTransformer', function() {
         },
         remediations: [
           {
-            action: jasmine.any(Function),
+            action: expect.any(Function),
```
Comment (TianGan-okta) [inline test/unit/spec/v2/ion/responseTransformer_spec.js:74]: 💯

**Review by TianGan-okta — APPROVED** [review]
LGTM 🌟

**Review by mauriciocastillosilva-okta — APPROVED** [review]

**Pull request closed without merging**